assets/aml-benchmark/tests/test_inference_postprocessor.py

```diff
@@ -0,0 +1,157 @@
+# ---------------------------------------------------------

```

Comment:
All of the comments from pre-processor test apply here as well.

Reply:
Done. Thanks.